auto-update.php

```diff
@@ -75,23 +84,30 @@ public function modify_transient( $transient ) {
 	 */
 	public function plugin_popup( $result, $action, $args ) {
 		if ( ! empty( $args->slug ) ) {
-			if ( $args->slug === current( explode( '/' , $this->file ) ) ) {
+			if ( 'wc-api-dev' === $args->slug ) {
 				$this->maybe_fetch_github_response();
-				$plugin = get_plugin_data( dirname( dirname( __FILE__ ) ) . '/' . $this->file );
-				$plugin = array(
-					'name'              => $plugin['Name'],
+				$plugin_data = get_plugin_data( dirname( dirname( __FILE__ ) ) . '/' . $this->file );
+
+				// Make sure our GitHub responses are present. If not, we can still show info from the plugin README below.
+				$tag_name = ! empty( $this->github_response['tag_name'] ) ? $this->github_response['tag_name'] : '';
+				$published_at = ! empty( $this->github_response['published_at'] ) ? $this->github_response['published_at'] : '';
+				$body = ! empty( $this->github_response['body'] ) ? $this->github_response['body'] : '';
+				$zipball_url = ! empty( $this->github_response['zipball_url'] ) ? $this->github_response['zipball_url'] : '';
```

Comment:
@jeffstieler "An exception would be using ! empty(), as testing for false here is generally more intuitive.".